templates/manage/email_accounts.tpl.html

```diff
@@ -91,6 +79,18 @@
       toggleFolderField();
       $('#email_account_form').submit(validateForm);
       $('.select_all').click(function() { Eventum.toggleCheckAll('items[]'); });
+      $('#testSettings').click(function(event) {
+           event.preventDefault();
+           $('#full_width_messages').removeClass().html('Please wait.. testing connection.').addClass('info');
+           var serializedData = $('#email_account_form').serialize();
+           $.ajax({
+               type: "POST",
+               url: "check_email_settings.php",
```

Comment:
i think we should create common entry point for ajax calls. there are too many of them, and each one does things by it's own, not sure which is even correct one. i.e how to properly check for authenticated user and role.

see here:
https://github.com/eventum/eventum/tree/v3.0.5/htdocs/ajax

they all do the authenticity check differently.